### ISSUE TITLE
Lost connection - please try to refresh

### ISSUE THREAD

**Issue by ChriZathens** [pyload/android-app#20]
Hi guys!
I keep getting this error when trying to connect to my pyload installation
It was working for years. I recently changed my server (It was an ubuntu 14.04 LTS now it is an 16.04)
But the new server has the same IP address as the older one
All port forwarding setting are correct on my firewall.
In any case, however, the android client cannot even connect using the internal address from inside my network
The backend port is the default (7227) and the machine is listening normally to port 7227:
~# netstat -an | grep 7227 | grep -i listen
tcp        0      0 0.0.0.0:7227            0.0.0.0:*               LISTEN
I also tried a different one (7777) without any luck
Any ideas?
My android version is 7.1.2 (Nexus 5x)

**Comment by ChriZathens** [comment]
Continuing from where I left it hehe..
I am still facing the same issue.. 

Downloaded the latest version which supports pyload 0.4.20
Installed on my Samsung Galaxy S9 (Android 10)
The backend port is 7227 and NAT works properly 
I keep getting the same error
I have pyload (0.4.20) running without any issue under the same dynamic dns address that fails on the android app

**Comment by GammaC0de** [comment]
> In any case, however, the android client cannot even connect using the internal address from inside my network

Do you still cannot connect using the internal address from inside the network?

**Comment by ChriZathens** [comment]
Yes, tested it last night.. Even using the internal address I cannot connect from inside the local lan.

**Comment by Chaprnks** [comment]
Having the same exact issue. I found another github issue on the main pyload repo. I changed the "ip host" & "ip listenaddr" to 0.0.0.0 (apparently thats the correct values). Now I get the issue with Wrong Username/Password, even when I create a second account

**Comment by OptimusGREEN** [comment]
Just trying app for first time and seem to have same issue as above. 
android v11
pyload v0.4.20
tried on lan using ip
tried using ddns
cant connect.
